src/builtins/danger_zone.rs

```diff
@@ -97,6 +97,7 @@ impl RuleSet for ForkAndExec {
              Sysno::execve, Sysno::execveat,
              Sysno::wait4, Sysno::waitid,
              Sysno::clone, Sysno::clone3,
+             Sysno::pipe, Sysno::pipe2
```

Comment:
Ah sorry I was unclear in my comment on the original issue.

You should make a separate method in this ruleset (or make a separate ruleset that's just called Pipes or something) which enables pipes separately from the rest. 

I think maybe just making a
```
pub struct Pipes; 
impl RuleSet for Pipes {
  [...] simple_rules [...] 
    vec![Sysno::pipe, Sysno::pipe2]
```

is probably the simplest.

Reply:
Thanks for the clarification. I'll adjust my PR accordingly